core/src/mindustry/ui/dialogs/PlanetDialog.java

```diff
@@ -412,6 +412,10 @@ void setup(){
                     public void tap(InputEvent event, float x, float y, int count, KeyCode button){
                         if(showing()) return;
 
+                        if(selected == hovered){
```

Comment:
This should check for a double-tap. If I click a sector once, then click it once again later, it should not send me there.